### PR TITLE
Associate user with legislator

### PR DESCRIPTION
**Pull request by andrewdwooten**
Establishes a many-to-many relationship between users and legislators via an addition to the Legislator model in models.py. Unit test added to test.py which confirms desired functionality of the relationship. I still have two request tests failing locally, but I think that's status quo for my machine. If someone could run the test suite from this branch to confirm tests are still passing before merging, that would be ideal.

### CODE REVIEW

File: public_officials/models.py [hunk 1]
```diff
@@ -17,6 +18,8 @@ class Legislator(models.Model):
     twitter_id = models.CharField(max_length=15, null=True)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    users = models.ManyToManyField(User)
```
Comment (TheKaterTot) [inline public_officials/models.py]: This is way simpler than what we had earlier, but still so similar. Nice catch!

File: public_officials/tests.py [hunk 2]
```diff
@@ -286,3 +287,42 @@ def test_recent_bills_url_resolves_to_bills_index_view(self):
...
+                                                   state="TX",
+                                                   cid="N00006134")
+        user = User.objects.create(username='test')
+        user2 = User.objects.create(username='test2')
+
+        legislator.users.add(user)
+        legislator.users.add(user2)
+        legislator_two.users.add(user)
+
+        self.assertEqual(legislator.users.count(), 2)
```
Comment (TheKaterTot) [inline public_officials/tests.py]: Nice tests! We need the tests right now more than ever to help us know what is happening. 👍

File: public_officials/migrations/0006_user.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('legislators', models.ManyToManyField(to='public_officials.Legislator')),
+            ],
+        ),
```
Comment (cmacaulay) [inline public_officials/migrations/0006_user.py]: 👏 👏 👏

File: public_officials/migrations/0007_auto_20170530_2234.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='user',
+            name='legislators',
+        ),
+        migrations.DeleteModel(
+            name='User',
+        ),
```
Comment (cmacaulay) [inline public_officials/migrations/0007_auto_20170530_2234.py]: great job figuring this out!

File: public_officials/models.py (hunk 1 above)
Comment (cmacaulay) [inline public_officials/models.py]: Thanks, Django! 👍 ⚡️

File: public_officials/templatetags/debug.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+from django.template import Library, Node
+
+register = Library()
+
+class PdbNode(Node):
+
+    def render(self, context):
+        pdb_module.set_trace()
+        return ''
```
Comment (cmacaulay) [inline public_officials/templatetags/debug.py]: 👏 🎉 🎉 🕺

File: public_officials/tests.py [hunk 6]
```diff
@@ -3,6 +3,7 @@
 from django.core.urlresolvers import resolve
 from public_officials.views import *
 from public_officials.models import *
+from django.contrib.auth.models import User
```
Comment (cmacaulay) [inline public_officials/tests.py]: @andrewdwooten is this bringing in the auth user model? Is this the same as our user model? Thanks for explaining 🏅 🤴

**Pull request merged**